src/Lykke.Job.HistoryExportBuilder.AzureRepositories/ExpiriesRepository/ExpiryEntryEntity.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Lykke.AzureStorage.Tables;
+using Lykke.Job.HistoryExportBuilder.Core.Domain;
+
+namespace Lykke.Job.HistoryExportBuilder.AzureRepositories.ExpiriesRepository
+{
+    public class ExpiryEntryEntity : AzureTableEntity, IExpiryEntry
+    {
+        public string ClientId { get; set; }
+        public string RequestId { get; set; }
+        public DateTime ExpiryDateTime { get; set; }
+
+        public static string GeneratePartitionKey()
+        {
+            return "part";
```

Comment:
Consider usage of several partiotions (for example - by clientId), cause 1 partition will lead to performance degradation in the long run